manifests/p/pierogii/EATEOF/6932a/pierogii.EATEOF.installer.yaml

```diff
@@ -0,0 +1,16 @@
+# Created using wingetcreate 1.2.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: pierogii.EATEOF
+PackageVersion: 6932a
+Installers:
+- InstallerUrl: https://files.gamebanana.com/mods/everywhere_at_the_end_of_funk_updat_6932a.zip
+  Architecture: x64
+  InstallerType: zip
+  NestedInstallerType: portable
+  NestedInstallerFiles:
+  - RelativeFilePath: Everywhere at the End of Funk Update\Funkin.exe
+    PortableCommandAlias: dementia-game

```

Comment:
```suggestion
  - PortableCommandAlias: dementia-game

```

Reply:
I'm not sure, but I believe this might be the cause of the manifest error.

Reply:
> I'm not sure, but I believe this might be the cause of the manifest error.

This is incorrect. It's because the `:` character isn't valid in YAML (See the locale manifest).